crates/ruma-client-api/src/media/get_content.rs

```diff
@@ -65,6 +65,14 @@ pub mod v3 {
             rename = "fi.mau.msc2246.max_stall_ms"
         )]
         pub timeout_ms: Duration,
+
+        /// Whether the server may return a 307 or 308 redirect response that points at the
+        /// relevant media content.
+        ///
+        /// When not explicitly set to `true`, the server must return the media content itself.
```

Comment:
```suggestion
        /// Unless explicitly set to `true`, the server must return the media content itself.
```

Maybe a bit more elegant? Same for the other files.